_topic_maps/_topic_map.yml

```diff
@@ -3191,8 +3191,6 @@ Topics:
 #- Name: Get started
 #  File: serverless-getting-started
 # uncomment once new content is added for this
-- Name: Update
-  File: serverless-update
```

Comment:
Removed this since it just tells users to update the channel to `stable`, but this has been the only available channel for some time now. The Operator updates automatically AFAIK.